client/db/src/lib.rs

```diff
@@ -1752,6 +1769,13 @@ impl<Block: BlockT> Backend<Block> {
 			columns::BODY,
 			id,
 		)?;
+		utils::remove_from_db(
+			transaction,
+			&*self.storage.db,
+			columns::KEY_LOOKUP,
+			columns::JUSTIFICATIONS,
+			id,
+		)?;
```

Comment:
May be worth mentioning in release note that justification are now pruned with block (additionally to the new parameters).
But looks like a good thing to me :+1: